scripts/cluster/test_cluster.sh

```
@@ -0,0 +1,281 @@
+    
+    local success_count=0
+    local total=5
+    
+    for i in $(seq 1 $total); do
+        local key="multi_test_$$_$i"
+        local value="value_$i"
+        
+        redis-cli -p $LEADER_REDIS_PORT SET "$key" "$value" > /dev/null 2>&1
+        
+        local follower_port=${REDIS_PORTS[1]}
+        local result=$(redis-cli -p $follower_port GET "$key" 2>/dev/null)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)